pyproject.toml

```
@@ -0,0 +1,89 @@
+[tool.poetry]
+name = "orchestration"
+version = "0.1.0dev"
```

```suggestion
version = "0.1.0"
```
I didn't chase the problem, but this translates into a `0.1.0.dev` (note the extra dot) in `get-orchestration-local-dist`. So the grep function doesn't work. Sticking with a numeric version works.